ledger/participant-integration-api/src/main/scala/platform/apiserver/AuthServiceConfig.scala

```diff
@@ -15,6 +15,7 @@ import com.daml.ledger.api.auth.{AuthService, AuthServiceJWT, AuthServiceWildcar
 
 sealed trait AuthServiceConfig {
   def create(): AuthService
+  val jwtTimestampLeeway: Option[JwtTimestampLeeway] = None
```

Comment:
I know I suggested this `val`, so I am guilty of this. But now upon more careful inspection, I'd rather we converted it to `def` as a `trait` should never define a `val` as a general rule.
```suggestion
  def jwtTimestampLeeway: Option[JwtTimestampLeeway]
```